### ISSUE TITLE
Likes the point scoring system (from blog comments)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#238]
Erik Gustafsson responded to An insider look at WCAG 3.0:

I really like the way the new guidelines have scoring and not a black and white view of what is considered failing. I think it will make conformance more fun for site owners, striving for gold medal instead of just saying AA is enough and no more…

Great job!

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
